test/item_repository_test.rb

```diff
@@ -1,38 +1,36 @@
-require_relative 'test_helper'
-require_relative '../lib/item_repository'
+require_relative "test_helper"
+require_relative "../lib/item_repository"
 
 class ItemRepositoryTest < Minitest::Test
   attr_reader :items, :item_repository, :sales_engine
 
   def setup
-    @items = [{
-        id: 3,
-        name: 'Item',
-        description:  'Ea Voluptatum,Sunt officia eum qui molestiae.',
-        unit_price: '67076',
-        merchant_id: 1,
-        created_at: '2012-03-27 14:53:59 UTC',
-        updated_at: '2012-03-27 14:53:59 UTC'
-      },
-      {
-        id: 2,
-        name: 'Item',
-        description: 'Autem Minima,Cumque consequuntur ad.',
-        unit_price: '67076',
-        merchant_id: 1,
-        created_at: '2012-03-27 14:53:59 UTC',
-        updated_at: '2012-03-27 14:53:59 UTC'
-      },
-      {
-        id: 7,
-        name: 'Item',
-        description: 'Expedita Fuga,Fuga assumenda occaecati hic dolorem tenetur dolores nisi.',
-        unit_price: '31163',
-        merchant_id: 1,
-        created_at: '2012-03-27 14:53:59 UTC',
-        updated_at: '2012-03-27 14:53:59 UTC'
-      }
-      ]
+    @items = [
+               { id: 3,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.